advanced.matrix/src/main/java/learning/solutions/advanced/matrix/utils/ReinforcementLearnerUtil.java

```diff
@@ -265,22 +265,22 @@ public static RCell getBest(RCell rCell, RCell[][] grid, int cellWidth, int fram
             }
         }
 
-        System.out.println("------------------------------------------------ PRINTING THE MATRIX " +
-                "------------------------------------------------");
-        for (int i = size - 1; i >= 0; i--) {
-            System.out.println();
-            for (int j = 0; j < size; j++) {
-                System.out.print("|");
-                System.out.print(" ");
-                System.out.print(toReturn[i][j].getLatLongIdentifier().getIdentifier() + ", " + toReturn[i][j].getId
-                        () + "," + i + "-" + j);
-                System.out.print(" ");
-            }
-            System.out.print("|");
-            System.out.println();
-        }
-        System.out.println("------------------------------------------------ END OF MATRIX " +
-                "------------------------------------------------");
+//        System.out.println("------------------------------------------------ PRINTING THE MATRIX " +
+//                "------------------------------------------------");
+//        for (int i = size - 1; i >= 0; i--) {
+//            System.out.println();
+//            for (int j = 0; j < size; j++) {
+//                System.out.print("|");
+//                System.out.print(" ");
+//                System.out.print(toReturn[i][j].getLatLongIdentifier().getIdentifier() + ", " + toReturn[i][j].getId
+//                        () + "," + i + "-" + j);
+//                System.out.print(" ");
+//            }
+//            System.out.print("|");
+//            System.out.println();
+//        }
+//        System.out.println("------------------------------------------------ END OF MATRIX " +
+//                "------------------------------------------------");
 
         // Compute adjacency!
```

Comment:
Is there a function call missing here? Compute adjacency should be taken care of as part of the set up.